02-project-intro.Rmd

```diff
@@ -163,6 +163,13 @@ Now we have a good directory structure we will now place/save the data file in t
 >
 > We will load and inspect these data later.
 
+> ## Challenge 2 {.challenge}
+> It is usefull to get some general idea about the dataset, directly from the
```

Comment:
According to README 'These lesson materials are designed to be run after both the Shell and Git materials'. In this challenge I want people to use their knowledge of shell in new context.